pwa-devdocs/src/venia-pwa-concept/features/modular-components/index.md

```diff
@@ -78,13 +78,37 @@ The Venia Adapter wraps around Venia components to satisfy any implicit external
 
 ### Venia drivers
 
+The [`@magento/venia-drivers`][] dependency is a _virtual module_ for centralizing the external dependencies of Venia components that use them, such as GraphQL clients and Redux stores.
+Instead of importing these dependencies directly, Venia components import them from `src/drivers`.
```

Comment:
Is this repeating L92 & L93?

Reply:
Can we drop L92 & L93?

Reply:
Since @jcalcaben wrote L92 and L93 I'd rather drop these lines, which I've done.